src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/NuGetUI.cs

```diff
@@ -306,7 +328,14 @@ public void EndOperation()
 
         public bool ForceRemove { get; set; }
 
-        public PackageIdentity SelectedPackage { get; set; }
+        public string SelectedPackageId
```

Comment:
I don't think this property was being used. However, I prefer that this parent model be responsible for knowing the selected Package ID, so I'm using it in this PR. 
I think future refactoring can be done to use it instead of digging up the Details Pane model to find the same value.